code/datums/abilities/ability_parent.dm

```diff
@@ -890,6 +890,9 @@
 			var/result = tryCast(target, params)
 #ifdef NO_COOLDOWNS
 			result = TRUE
+			var/result = TRUE
+#else
+			var/result = tryCast(target, params)
```

Comment:
Errors are coming from this not being reverted.

Reply:
thanks! much appreciated